xilem_core/Cargo.toml

```diff
@@ -1,21 +1,16 @@
 [package]
 name = "xilem_core"
 version = "0.1.0"
-description = "Common core of the Xilem Rust UI framework."
-keywords = ["xilem", "ui", "reactive", "performance"]
-categories = ["gui"]
-publish = false # Until it's ready
 edition.workspace = true
 license.workspace = true
 repository.workspace = true
-homepage.workspace = true
 
-[package.metadata.docs.rs]
```

Comment:
We should recover this metadata

We probably need to have a standard set of this documented (on the wiki?)